app/guide/[category].tsx

```
@@ -95,7 +98,7 @@ const CategoryScreen = () => {
 
         <FlatList
           style={styles.list}
-          contentContainerStyle={styles.listContent}
+          contentContainerStyle={[styles.listContent, { paddingBottom: insets.bottom + layout.screenPaddingBottom }]}
```

ESLint

[error] Replace `styles.listContent,·{·paddingBottom:·insets.bottom·+·layout.screenPaddingBottom·}` with `⏎············styles.listContent,⏎············{·paddingBottom:·insets.bottom·+·layout.screenPaddingBottom·},⏎··········`

(prettier/prettier)